src/packages/migrate/src/utils/seed.ts

```diff
@@ -0,0 +1,72 @@
+import fs from 'fs'
+import path from 'path'
+import execa from 'execa'
+
+export function detectSeedFiles() {
+  const seedPath = path.join(process.cwd(), 'prisma', 'seed.')
+
+  const jsFilePath = seedPath + 'js'
+  const tsFilePath = seedPath + 'ts'
+  const shFilePath = seedPath + 'sh'
+  const goFilePath = seedPath + 'go'
+
+  const detected = {
+    seedPath,
+    numberOfSeedFiles: 0,
+    js: fs.existsSync(jsFilePath) ? jsFilePath : null,
+    ts: fs.existsSync(tsFilePath) ? tsFilePath : null,
+    sh: fs.existsSync(shFilePath) ? shFilePath : null,
+    go: fs.existsSync(goFilePath) ? goFilePath : null,
+  }
+
+  detected.numberOfSeedFiles =
+    Number(!!detected.js) +
+    Number(!!detected.ts) +
+    Number(!!detected.sh) +
+    Number(!!detected.go)
```

Comment:
how about:

```suggestion
  const detected = {
    seedPath,
    numberOfSeedFiles: 0,
  }
  
  const paths = ['js', 'ts', 'sh', 'go']

  for (const path of paths) {
    if (!fs.existsSync(jsFilePath) {
      continue
    }
    const fullPath = seedPath + path
    detected[path] = fullPath
    detected.numberOfSeedFiles++
  }
```

Reply:
Done in a commit with small changes, Thanks!